### PR TITLE
Frontend: added a view to add new contributors on Team page

### PR DESCRIPTION
**Pull request by artkorenev**
Resolving #891 added a view. Provided implementation without "Headshot" and "Background images" inputs because no file upload has been implemented yet.

Preview of the view:
image

### CODE REVIEW

**Comment by deshraj** [comment]
@artkorenev: Can you please add the functionality of uploading the files?

**Comment by artkorenev** [comment]
@deshraj is it currently implemented within the project?

**Comment by deshraj** [comment]
@artkorenev yeah it is there in the challenge submission view. Feel free to borrow code from there.

File: frontend/src/js/controllers/ourteamCtrl.js [hunk 1]
```diff
@@ -8,43 +8,104 @@
...
+
+        vm.loadTeamList = function() {
+            var parameters = {};
+            parameters.url = 'web/team/';
+            parameters.method = 'GET';
+            parameters.callback = {
+                onSuccess: function(response) {
+                    var status = response.status;
+                    var results = response.data;
+                    if (status == 200) {
```
Comment (aayusharora) [inline frontend/src/js/controllers/ourteamCtrl.js]: Can you please Replace this with `===`

File: frontend/src/js/controllers/ourteamCtrl.js [hunk 2]
```diff
@@ -8,43 +8,104 @@
...
+                "email": vm.contributor.email,
+                "headshot": vm.contributor.headshot,
+                "github_url": vm.contributor.github_url,
+                "linkedin_url": vm.contributor.linkedin_url,
+                "personal_website": vm.contributor.personal_website,
+                "background_image": vm.contributor.background_image
+            };
+
+//            TODO: add headshot and background_image upload
+            if (vm.contributor.headshot) {
```
Comment (aayusharora) [inline frontend/src/js/controllers/ourteamCtrl.js]: Empty If statement? It is not needed, I guess.

File: frontend/src/js/controllers/ourteamCtrl.js [hunk 3]
```diff
@@ -8,43 +8,104 @@
...
+                onSuccess: function() {
+                    $rootScope.notify("success", "Contributor '" + vm.contributor.name + "' has been added successfully!");
+                    // clearing the input form
+                    vm.contributor = {};
+                    // reloading the teams from server after a successful submission
+                    vm.loadTeamList();
+                },
+                onError: function(response) {
+                    var error = response.data;
+                    if (response.status == 400) {
```
Comment (aayusharora) [inline frontend/src/js/controllers/ourteamCtrl.js]: `===` here

File: frontend/src/views/web/our-team.html [hunk 4]
```diff
@@ -67,6 +67,83 @@ <h4 class="team-heading center-align">{{ourTeam.contributingTeamType}}</h4>
                     </div>
                 </div>
             </div>
+            <!-- Section for adding new contributors -->
+            <div class="row">
+                <div class="col s12 offset-m2 m8 ev-mt-50">
+                    <h4 class="text-dark-black align-center"><strong>Add a contributor</strong></h4>
```
Comment (aayusharora) [inline frontend/src/views/web/our-team.html]: Please avoid using `<strong>` and try to achieve the font-weight by `css`

File: frontend/src/js/controllers/ourteamCtrl.js (hunk 2 above)
Comment (aka-jain) [inline frontend/src/js/controllers/ourteamCtrl.js]: I think that's in Todo list, read comment.

**Comment by deshraj** [comment]
Seems to be inactive from a long time. Hence closing this one.

**Pull request closed without merging**